awwvision/src/main/java/com/google/cloud/servicebroker/samples/awwvision/service/CuteImageService.java

```diff
@@ -34,19 +34,19 @@
 import java.util.Map;
 
 /**
- * Helper methods for interacting with the Cloud Storage API.
+ * Component which handles the storage of cuteimages from https://reddit.com/r/aww
  *
  * <p>Uses the Cloud Storage Bucket configured in the application properties.
  */
 @Controller
-public class StorageApi {
+public class CuteImageService {
 
   @Autowired
   private Storage storageService;
 
   private final String bucketName;
 
-  StorageApi() {
+  CuteImageService() {
     String env = System.getenv("VCAP_SERVICES");
```

Comment:
I'm not sure if this is needed if we have the right spring-boot-starter, I think that one is set up to work with storage buckets, right?

Reply:
Yup, I was going to do that in another commit. I'll add it to this PR